recipes/geoviews-core/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "geoviews-core" %}
+{% set version = "1.5.1" %}
+{% set sha256 = "e2fda0fc0e09175a2de72dca018bcaca78fa9ca0de782ed7e243a95ee90b4b43" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/geoviews/geoviews-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - geoviews = geoviews.__main__:main
+
+requirements:
+  build:
+    - python
+    - pip
+    - param >=1.6.1,<2.0
+  run:
+    - python >=2.7
+    - bokeh >=0.12.15
+    - cartopy >=0.14.2
+    - holoviews >=1.10.1
+    - numpy >=1.0
+    - param >=1.6.1,<2.0
+
+test:
+  imports:
+    - geoviews
+  requires:
+    - nose
+  commands:
+    - nosetests geoviews
+
+about:
+  home: https://geoviews.org
+  license: BSD 3-Clause
+  license_file: LICENSE
+  summary: GeoViews is a Python library that makes it easy to explore and visualize
+        geographical, meteorological, and oceanographic datasets, such as those used
+        in weather, climate, and remote sensing research.
+
+extra:
+  recipe-maintainers:
+    - ocefpaf
+    - philippjfr
```

Comment:
Do you want to add yourself as a maintainer?
(We can do that in the feedstock, merging this to unblock https://github.com/conda-forge/geoviews-feedstock/pull/4)